### PR TITLE
(DOCSP-22380): remove unneeded waitForSynchronization + non awaited subs.update

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-22380

### Staged Changes (Requires MongoDB Corp SSO)

- [Flexible Sync - React Native SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/waitForSynchronization-notNeeded-JS/sdk/react-native/examples/flexible-sync/)
- [Flexible Sync - Node.js SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/waitForSynchronization-notNeeded-JS/sdk/node/examples/flexible-sync/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/node/examples/flexible-sync.txt [hunk 1]
```diff
@@ -142,27 +142,10 @@ Add queries to your subscriptions list to sync the query's data. To do so, perfo
    :js-sdk:`add() <Realm.App.Sync.MutableSubscriptionSet.html#add>`
    method on the ``MutableSubscriptionSet`` to add a query to the subscription.
 
-.. tabs::
+The following example shows how to subscribe to the queries created earlier.
```
Comment (MongoCaleb) [inline source/sdk/node/examples/flexible-sync.txt:145]: "earlier" or "above"?

File: source/sdk/react-native/examples/flexible-sync.txt [hunk 2]
```diff
@@ -142,28 +142,11 @@ Add queries to your subscriptions list to sync the query's data. To do so, perfo
...
-   .. tab:: Synchronously Add a Query
-      :tabid: synchronous-add-a-query-to-subscriptions
-
-      You can synchronously add a single query or multiple queries within a ``subscriptions.update()``
-      transaction. If you synchronously add queries, you must call the :ref:`waitForSynchronization
-      <react-native-flexible-sync-wait-for-sync>` method afterward.
-      
-      .. literalinclude:: /examples/generated/node/flexible-sync.codeblock.subscribe-to-queryable-fields.js
-          :language: javascript      
+The following example shows how to subscribe to the queries created earlier.
```
Comment (MongoCaleb) [inline source/sdk/react-native/examples/flexible-sync.txt:145]: same as above. Not sure if "earlier" is the right word.

**Review by MongoCaleb — APPROVED** [review]
LGTM % one word question

File: source/sdk/node/examples/flexible-sync.txt (hunk 1 above)
Comment (mongomoe) [inline source/sdk/node/examples/flexible-sync.txt:145]:
```suggestion
The following example shows how to subscribe to the queries created above.
```

File: source/sdk/react-native/examples/flexible-sync.txt (hunk 2 above)
Comment (mongomoe) [inline source/sdk/react-native/examples/flexible-sync.txt:145]:
```suggestion
The following example shows how to subscribe to the queries created above.
```

File: source/sdk/node/examples/flexible-sync.txt (hunk 1 above)
Comment (mongomoe) [reply source/sdk/node/examples/flexible-sync.txt]: fair enough, changed to "above" 👍

**Pull request merged**